doc/getting-started/install.md

```diff
@@ -132,13 +92,33 @@ make
 sudo make install
 ```
 
-Add the following to your .bashrc file and source it:
+Add the following to your `~/.bashrc` file and source it (or re-open the terminal):
 
 ```bash
 export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
 export PKG_CONFIG_PATH="/usr/local/lib/pkgconfig:$PKG_CONFIG_PATH"
 ```
 
+#### Installing Secp256k1
+
+Create a working directory for your builds:
+
+```bash
+mkdir -p ~/src
+cd ~/src
+```
+
+Download and install `libsecp256k1`:
+
+```bash
+git clone https://github.com/bitcoin-core/secp256k1
+cd secp256k1
```

Comment:
Are we asking users to follow main branch instead of specific commit used in github workflows of this repo?

Reply:
Indeed this is a good point, and as I don't have the knowledge to say that master would be OK, I changed it to use the same commit as the github workflow. Thanks.